pandas/api/test.py

```diff
@@ -0,0 +1,23 @@
+"""
+Entrypoint for testing from the top-level namespace
+"""
+import os
+
+PKG = os.path.dirname(os.path.dirname(__file__))
```

Comment:
That would, but I think then pytest discovers "tests" in vb_suite, numpydoc, and scripts.